cloudformation/media-atom-maker.json

```diff
@@ -345,17 +344,16 @@
                                         ]
                                     ]
                                 },
-                                "aws s3 cp 's3://atom-maker-conf/media-atom-maker.",
+                                "/opt/features/native-packager/install.sh -b atom-maker-dist -a media-atom-maker -t tgz\n",
+
+                                "mkdir /etc/gu \n",
+                                "aws s3 cp 's3://atom-maker-conf/",
                                 {
-                                    "Ref": "Stage"
+                                  "Ref": "Stage"
                                 },
-                                ".application.conf' ",
-                                "/home/media-service/media-atom-maker/conf/application.conf\n",
-                                "cat >> /home/media-service/media-atom-maker/conf/application.conf<<'EOF'\n",
-                                "aws.dynamo.tableName=", { "Ref": "AtomMakerTable" }, "\n",
```

Comment:
I think it's the same for both, you'd have to recycle the instance to get the new dynamo config? 
I'll remove the params from the cloudformation as they won't be used anymore.